schema.sql

```diff
@@ -17,6 +17,8 @@ WEIGHT_KG FLOAT
 );
 
 
+-- DAY 2
+ALTER TABLE ANIMALS ADD SPECIES TEXT;
```

Comment:
- [OPTIONAL] Using a `Varchar` would be ideal here because you can store a certain amount of character.  But for `TEXT` it has a fixed size. Please read[ this article](https://www.navicat.com/en/company/aboutus/blog/1308-choosing-between-varchar-and-text-in-mysql#:~:text=Some%20Differences%20Between%20VARCHAR%20and%20TEXT&text=The%20VAR%20in%20VARCHAR%20means,be%20part%20of%20an%20index.)

Reply:
I Will take this into account , thank you